### ISSUE TITLE
IPBan Crash

### ISSUE THREAD

**Issue by RvdHout** [DigitalRuby/IPBan#49]
I have noticed a crash of IPBan in my logfile, log rotation? (version v1.3.6)
Or maybe due to a restart of the system? ...i think i restarted this machine after installing Windows Updates around 08:30 this morning

```
2019-01-10 08:30:59.8496|ERROR|IPBan.IPBanLog|Exception: System.IO.IOException: The process cannot access the file 'C:\Windows\TEMP\tmpD41F.tmp' because it is being used by another process.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize)
   at System.IO.StreamReader..ctor(String path, Encoding encoding)
   at System.IO.File.InternalReadAllLines(String path, Encoding encoding)
   at System.IO.File.ReadAllLines(String path)
   at IPBan.IPBanOS..cctor() in C:\Users\Jeff\Documents\GitHub\Windows-IP-Ban-Service\Core\IPBanOS.cs:line 102
```

**Comment by jjxtra** [comment]
I checked in a fix (hopefully) for this, are you able to build from source?

**Comment by jjxtra** [comment]
Should be fixed, I refreshed the 1.3.6 release, it will report any problem obtaining the os version.
